include/image_publisher.h

```diff
@@ -11,6 +11,7 @@
 
 #include <ros/console.h>
 #include <camera_info_manager/camera_info_manager.h>
+#include <highgui.h>
```

Comment:
why include `highgui` ? 

(sorry just looking at the new review in github :D )